packages/backend/apps/users/management/commands/init_customers_plans.py

```diff
@@ -0,0 +1,20 @@
+from apps.finances.services import subscriptions
+from django.core.management.base import BaseCommand
+
+from config import settings
+from ...models import User
+
+
+class Command(BaseCommand):
+    help = 'Creates stripe customer and schedule subscription plan'
+
+    def handle(self, *args, **options):
+        if not settings.STRIPE_LIVE_MODE and not settings.STRIPE_TEST_SECRET_KEY:
+            return
+
+        if settings.STRIPE_LIVE_MODE and not settings.STRIPE_LIVE_SECRET_KEY:
+            return
```

Comment:
what do you think of this line and the above if statement to log those instances where keys are missing? Django commands from what I remember have built-in support for printing things and verbose flags (see self.stdout for django commands classes)

Reply:
I've checked that and actually we don't need those checks at all. There are built in checks in djstripe lib, which are checking for stripe keys and it's format, so if keys are not present or malformed it won't start the server.
I'll remove them from this command and init_subscriptions as well